README.md

```diff
@@ -8,16 +51,238 @@ so don't try to use it for anything close to a production environment.
 
 ***Note: this README is only for development purposes.***
 
-## Quick Start
 
-### Build binary
+## <a name='OperatorDeployment'></a>Operator Deployment
+
+The ORAN O2 IMS implementation in OpenShift is managed by the IMS operator. 
+It configures the different components defined in the specification: 
+the deployment managers service, the resource server, alert server, subscriptions to resource and alert.
+
+The IMS operator will create an O-Cloud API that will be available to be queried, for instance from a SMO. It
+also provides a configuration mechanism using a Kubernetes custom resource definition (CRD) that allows the hub cluster administrator to configure the different IMS microservices properly.
+
+
+### <a name='Deploytheoperatoronyourcluster'></a>Deploy the operator on your cluster
+
+The IMS operator is installed on an OpenShift cluster where Red Hat Advanced Cluster Management for Kubernetes (RHACM), a.k.a. hub cluster, is installed too. Let’s install the operator. You can use the latest automatic build from the [openshift-kni namespace](https://quay.io/repository/openshift-kni/oran-o2ims-operator) in quay.io or build your container image with the latest code. 
+
+If you want to build the image yourself and push it to your registry right after:
+
+> :warning: Replace the IMAGE_NAME value with the full name of your container image
+
+```bash
+$ export IMAGE_NAME=quay.io/alosadag/oran-o2ims:v4.16
```

Comment:
Since this README is for development purposes the version used here should be `latest` to ensure that the right image is always used for re-deployments.

Reply:
This is an example of deployment using an image built by the user. See that it is alosadagrande/oran-o2ims:tag.  

If this is for development, there must be quite a few occasions when the developer/user has to build its own image, either with the new changes he/she introduced (test) or because the "official" image does not have them yet.

Reply:
Yes, I just meant to change the `v4.16` to `latest`... since that's the typical dev workflow.